WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -617,6 +617,45 @@
         expect(charge.metadata.competition).to eq competition.name
       end
 
+      it 'processes payment with donation and valid credit card' do
```

Comment:
This case is pretty much the same as the previous one, with an amount higher than the registration fees.
Any elegant way to avoid code duplication here? Should I keep only one (preferably the one creating a donation)?